### DISCUSSION TITLE
Request QR Code Tidak Tampil

### DISCUSSION THREAD

**Discussion by tes-kious** [kirimwa-id/forum#13] [Tanya Jawab]
ketika saya akses URL :
https://api.kirimwa.id/v1/qr?device_id=Redmi 4A
respon dari sistem adalah 
 { "message": "404 not found." }

**Comment by rioastamal** [comment]
Halo @tes-kious, Perintah yang digunakan seperti apa? Pertanyaan anda kurang jelas. Silahkan ikuti panduan membuat pertanyaan di https://github.com/kirimwa-id/forum/discussions/2

**Reply by tes-kious** [reply]
ketika akses 
curl 'https://api.kirimwa.id/v1/qr?device_id=iphone-x-pro'
  -H 'Authorization: Bearer YOUR_API_TOKEN'

responsenya :
{ "message": "404 not found." }

mohon info ada kesalahan dimana

**Reply by rioastamal** [reply]
error 404 not found hanya terjadi jika endpoint salah. Sedangkan end point `https://api.kirimwa.id/v1/qr` ada. Silahkan kirimkan link video atau gambar saja biar lebih jelas. Tuliskan juga OS (Windows, macOS, Linux) yang digunakan.
